tensorflow/core/framework/shape_inference.cc

```diff
@@ -1099,14 +1104,16 @@ Status InferenceContext::AttachContext(const Status& status) {
   std::vector<string> input_from_tensors_str;
   std::vector<string> input_from_tensors_as_shape_str;
   input_from_tensors_as_shape_str.reserve(inputs_.size());
-  for (int i = 0; i < inputs_.size(); ++i) {
+  for (int i = 0, iter_limit = inputs_.size(); i < iter_limit; ++i) {
+    const int input_tensors_as_shapes_size_ = input_tensors_as_shapes_.size();
+    const int input_tensors_size_ = input_tensors_.size();
```

Comment:
Coding style does not like local variables with name ending in `_`.

Reply:
Don't forget about this one

Reply:
Didn't realize I added the ending `_`, thought I removed it.

Reply:
You have `_` at the end again.